src/components/FiltersCollection.jsx

```diff
@@ -0,0 +1,66 @@
+import '../styles/search.css'
+import React, { useState } from 'react'
+import { applySearchNameAsync, getAllItemsAsync } from '../redux/items/thunks'
+import { useDispatch } from 'react-redux'
+import { Button } from 'react-bootstrap'
+
+function FiltersCollection () {
+  const dispatch = useDispatch()
+  const [searchInput, setSearchInput] = useState('')
+
+  const handleClearSearch = () => {
+    setSearchInput('')
+  }
+
+  const handleApplySearch = () => {
+	if (searchInput === '') {
+		dispatch(getAllItemsAsync())
+	} else {
+		dispatch(applySearchNameAsync(searchInput))
+	}
+	setSearchInput('')
```

Comment:
maybe the search input shouldn't be reset when the user searches? e.g. when you search on Google your input stays in the search bar.